test/integration/inject/inject_test.go

```diff
@@ -428,8 +428,7 @@ func TestInjectAutoPod(t *testing.T) {
 				Add: []v1.Capability{v1.Capability("NET_ADMIN"), v1.Capability("NET_RAW")},
 			},
 			Privileged:               &falsy,
-			RunAsUser:                &zero,
```

Comment:
Can you remove the `zero` var declaration right above? the linter is complaining about it.